lisa/tools/hwclock.py

```diff
@@ -22,8 +22,13 @@ def can_install(self) -> bool:
 
     def install(self) -> bool:
         posix_os: Posix = cast(Posix, self.node.os)
-        package_name = "util-linux"
-        posix_os.install_packages(package_name)
+        try:
+            package_name = "util-linux"
+            posix_os.install_packages(package_name)
+            self._check_exists()
+        except LisaException:
+            package_name = "util-linux-extra"
+            posix_os.install_packages(package_name)
```

Comment:
package_name = "util-linux"
            posix_os.install_packages(package_name)
            if not self._check_exists():
                package_name = "util-linux-extra"
                posix_os.install_packages(package_name)